features/in_onderzoek.feature

```diff
@@ -7,134 +7,738 @@ Functionaliteit: in onderzoek
   
   Een veld dat niet in onderzoek is, wordt niet in inOnderzoek opgenomen, ook niet met de waarde false of null.
 
-  De vertaling van een inOnderzoek elementwaarde (gegeven 83.10) naar inOnderzoek velden in het antwoord staat in /features/in onderzoek.xlsx
+  Een overzicht van de vertaling van een inOnderzoek elementwaarde (gegeven 83.10) naar inOnderzoek velden in het antwoord staat in /features/in onderzoek.xlsx
 
   Ook wanneer een gegeven geen waarde heeft en daardoor niet in het antwoord opgenomen wordt kan het in onderzoek zijn. In dat geval wordt alleen in inOnderzoek een veld opgenomen met die naam en de waarde true.
 
-Rule: wanneer een element in de bron in onderzoek is, wordt het gegeven in het antwoord dat daaruit gevuld wordt ook in inOnderzoek opgenomen met de waarde true
+  @proxy
+  Rule: wanneer een element in de bron in onderzoek is, wordt het gegeven in het antwoord dat daaruit gevuld wordt ook in inOnderzoek opgenomen met de waarde true
 
-  Abstract Scenario: adres heeft <veld> in onderzoek
+    Abstract Scenario: persoon heeft <veld> in onderzoek
+      Gegeven het systeem heeft een persoon met de volgende gegevens
+      | naam                        | waarde    |
+      | burgerservicenummer         | 555550001 |
+      En de persoon heeft de volgende 'inOnderzoek' gegevens
+      | naam                          | waarde   |
+      | aanduidingGegevensInOnderzoek | <waarde> |
+      | datumIngangOnderzoek          | 20220307 |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 555550001                       |
+      | fields              | <veld>                          |
+      Dan heeft de persoon met burgerservicenummer '555550001' de volgende 'inOnderzoek' gegevens
+      | naam                 | waarde     |
+      | <veld>               | true       |
+      | datumIngangOnderzoek | 2022-03-07 |
 
-    Voorbeelden:
-    | waarde | veld                    |
-    | 081120 | huisnummer              |
-    | 081130 | huisletter              |
-    | 081140 | huisnummertoevoeging    |
-    | 081150 | aanduidingBijHuisnummer |
-    | 081160 | postcode                |
-    | 081170 | woonplaats              |
-    | 081210 | locatiebeschrijving     |
+      Voorbeelden:
+      | waarde | veld                    |
+      | 010120 | burgerservicenummer     |
+      | 010410 | geslachtsaanduiding     |
 
-  Abstract Scenario: <in onderzoek> in onderzoek
+    Abstract Scenario: persoon heeft <groep> <veld> in onderzoek
```

Comment:
Wat is nu precies het verschil tussen dit abstracte scenario en dat van regel 64? Ze lijken me nl. gelijk alleen met andere voorbeelden.

Reply:
de ene gaat over inOnderzoek van de persoon, die uitwerking heeft in bepaalde groepen (naam en geboorte). De andere gaat over inOnderzoek van een andere categorie (verblijfplaats of gezagsverhouding). Het Gegeven deel van het scenario ziet er daardoor iets anders uit.